entities/reader.rb

```diff
@@ -0,0 +1,26 @@
+
+
+class Reader
+  attr_accessor :name, :email, :city, :street, :house
+  include Validation
+  include Errors
+
+  def initialize(**args)
+    @name = args[:name]
+    @email = args[:email]
+    @city = args[:city]
+    @street = args[:street]
+    @house = args[:house]
+    validation(@name, @email, @city, @street, @house)
+  end
+
+  private
+
+  def validation(name, email, city, street, house)
+    [name, email, city, street].map do |item|
```

Comment:
`map` creates a new array
you need `each` to iterate over each element in the array without creating an array